manager/manager.go

```diff
@@ -623,3 +625,28 @@ func (self *manager) watchForNewContainers(quit chan error) error {
 	}()
 	return nil
 }
+
+func (self *manager) watchForNewOoms() error {
+	outStream := make(chan *oomparser.OomInstance)
+	oomLog, err := oomparser.New()
+	if err != nil {
+		return err
+	}
+	go func() {
+		err := oomLog.StreamOoms(outStream)
+		glog.Infof("Streaming ooms in manager")
+		if err != nil {
+			glog.Errorf("%v", err)
```

Comment:
Actually, can we move this outside of the Go routine? We can just pass the channel to the Go routine.